migration/migration.go

```diff
@@ -0,0 +1,68 @@
+package migration
+
+import (
+	"encoding/binary"
+	"errors"
+
+	"github.com/NethermindEth/juno/db"
+)
+
+type revision func(db.DB) error
+
+var revisions = []revision{
+	revision0000,
+}
```

Comment:
Revisions list contains a set of revisions that can be applied to a database. Currently there is only one revision which makes sure the initial DB the node is using is empty.

Reply:
After making braking changes to the DB layout, we will have to add new revisions to this list.

Reply:
Please add the above explanation as comments